sys/src/cmd/stats.c

```diff
@@ -277,9 +277,12 @@ mkcol(int i, int c0, int c1, int c2)
 void
 colinit(void)
 {
-	mediumfont = openfont(display, "/lib/font/bit/pelm/latin1.8.font");
+
+	mediumfont = openfont(display, getenv("font"));
 	if(mediumfont == nil)
 		mediumfont = font;
+	else
+		mediumfont = openfont(display, "/lib/font/bit/go/Go-Mono/Go-Mono.14.font");
```

Comment:
We already set $font to Go-Mono.14.font in /usr/harvey/lib/profile, so that's what it should be getting. I think you should be doing this instead:

mediumfont = openfont(display, getenv("font"));
if (mediumfont == nil)
   mediumfont = openfont(display, "/lib/font/bit/pelm/latin1.8.font");
if (mediumfont == nil)
   mediumfont = font;

But yeah we should make this a library function, "opendefaultfont" or whatever that attempts to first open $font and then whatever the fallback is (pelm)